code/modules/clothing/glasses/_glasses.dm

```diff
@@ -200,6 +200,29 @@
 	icon_state = (icon_state == base_icon_state) ? "[base_icon_state]_flipped" : base_icon_state
 	user.update_worn_glasses()
 
+/obj/item/clothing/glasses/eyepatch/medical
+	name = "medical eyepatch"
+	desc = "Used by space weeaboos to pretend their eye isn't there, and crewmembers who actually lost their eye to pretend their eye is there."
+	icon_state = "eyepatch_medical"
+	base_icon_state = "eyepatch_medical"
+	inhand_icon_state = null
+
+/// wizard version
+/obj/item/clothing/glasses/eyepatch/medical/chuuni
```

Comment:
should the wizard version get flash protection so you don't end up being pushed to discard it in favor of sechuds in order to not instalose to borgs/flashbangs?

Reply:
you cant discard it its nodrop